.github/workflows/wheels.yml

```diff
@@ -152,7 +152,7 @@ jobs:
             echo "ANACONDA_UPLOAD=false" >> $GITHUB_ENV
           fi
       - name: Upload wheels
-        if: ${{ env.TOKEN }}
+        # if: ${{ env.TOKEN }}  # doesn't work
```

Comment:
Would it make more sense to just fuse the two steps ``Setup Upload Variables`` and ``Upload Wheels`` together?